django/core/management/commands/makemessages.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, dirpath, file_name, locale_dir):
         self.locale_dir = locale_dir
 
     def __repr__(self):
-        return "<TranslatableFile: %s>" % os.sep.join([self.dirpath, self.file])
+        return "<%s: %s>" % (self.__class__.__name__,
```

Comment:
We allow lines up to 119 characters if it helps readability, or we use hanging indent like this:
```
return "<%s: %s>" % (
    self.__class__.__name__,
    ...,
)
```